tests/correctness/resources/miniboxing/tests/compile/mb_array_04.check

```diff
@@ -1,3 +1,9 @@
+newSource1.scala:8: warning: Use MbArray instead of Array and benefit from miniboxing specialization
+  val mba1 = MbArray.clone(new Array[Int](10))
+                           ^
+newSource1.scala:9: warning: Use MbArray instead of Array and benefit from miniboxing specialization
+  val mba2 = MbArray.clone(new Array[Double](10))
+                           ^
```

Comment:
It's a two-way thing -- you guys motivate me to answer as quickly as I can and do my best, most likely because you do so yourselves :+1: